pkg/util/templates/templates.go

```diff
@@ -123,9 +122,9 @@ func (r *multiWindowMultiBurnAlertRule) render(url, percent, label, alertName st
 }
 
 // TemplateForPrometheusRuleResource returns a PrometheusRule
-func TemplateForPrometheusRuleResource(url, percent string, namespacedName types.NamespacedName) monitoringv1.PrometheusRule {
+func TemplateForPrometheusRuleResource(url, percent string, namespacedName types.NamespacedName, sourceCRName string) monitoringv1.PrometheusRule {
 
-	routeURLLabel := fmt.Sprintf(`RouteMonitorUrl="%s"`, url)
+	routeURLLabel := fmt.Sprintf(`%s="%s"`, sourceCRName, url)
```

Comment:
I was thinking on it, and this kind of label is not generic..
what do you think on `monitoredUrl`?

I am totally not strict on changing it now